cli/flags.rs

```diff
@@ -56,106 +74,114 @@ pub struct DenoFlags {
   // Use tokio::runtime::current_thread
   pub current_thread: bool,
 
+  pub bundle_output: Option<String>,
```

Comment:
It's strange that this is part of `DenoFlags` - it's very specific to `deno bundle`, why not put it into `argv`?

Reply:
I actually don't like the fact that so many subcommands use argv - I think it should be restricted to the "Run" subcommand.  I added some TODOs elsewhere in this patch that argv should become an `Option<Vec<String>>`. The worry is that it's being used as a general purpose way to pass data out.

The case of bundle, it does run JS code but has some bits of rust code to help it, so it's not a real Run command. I removed this bit from lib.rs where argv is being used inappropriately 
```rust
  assert!(state.argv.len() >= 2);	
  let out_file = match state.argv.len() {	
    3 => Some(state.argv[2].clone()),	
    _ => None,	
  };
```